tests/macros/groupdata.test.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * @prettier
+ */
+
+const { itMacro, describeMacro } = require('./utils');
+
+/*
+Different strings in GroupData objects have different sets of
+permitted characters.
+*/
+const permittedCharacters = {
+  group:      /^[\w ()-]+$/,
+  overview:   /^[\w ()-]+$/,
+  interface:  /^[\w.]+$/,
```

Comment:
Is it worth testing for the first character of an interface name being upper-case, per the rules for naming interfaces?

Reply:
Thanks @a2sheppy , I've made this change.